pkg/plugins/pluggable/plugins_integration_test.go

```diff
@@ -0,0 +1,58 @@
+// +build integration
+
+package pluggable
+
+import (
+	"os/exec"
+	"path"
+	"testing"
+
+	"get.porter.sh/porter/pkg/config"
+	"get.porter.sh/porter/pkg/secrets"
+	"github.com/stretchr/testify/require"
+)
+
+func TestPlugins_CatchStderr(t *testing.T) {
+	c := config.NewTestConfig(t)
```

Comment:
This runs the test in a temp directory instead of in your real porter home directory.

```suggestion
        home, err := ioutil.TempDir("/tmp", "porter")
	require.NoError(t, err, "could not create porter home directory")

	c := config.NewTestConfig(t)
	c.SetHomeDir(home)
```